initialization/routes.py

```diff
@@ -9,17 +9,19 @@
 import pages.upload as upload_pages
 import pages.submissions as submission_pages
 import pages.users as user_pages
+import pages.projects as project_pages
+import pages.comparison as comparison_pages
 
 
 def init_routes(app):
-    app.add_url_rule("/",      view_func=static_pages.static_home_page)
-    app.add_url_rule("/help",  view_func=static_pages.static_help_page)
-    app.add_url_rule("/about", view_func=static_pages.static_about_page)
+    app.add_url_rule("/",       view_func=static_pages.static_home_page)
+    app.add_url_rule("/help/",  view_func=static_pages.static_help_page)
+    app.add_url_rule("/about/", view_func=static_pages.static_about_page)
```

Comment:
There seem to be some issues if the URL opened ends with a slash. The recommended approach seems to be to add a slash to the route by default and Flask will make sure that opening `/help` and `/help/` both go to the same canonical URL, `/help/`.